packages/office-addin-manifest/src/commands.ts

```diff
@@ -149,6 +149,8 @@ export async function validate(manifestPath: string, command: commander.Command)
         }
       }
     }
+
+    process.exit(validation.isValid ? 0 : 1);
```

Comment:
Please have this set the exit code but not call process.exit(). It can exit normally. `office-addin-lint` has an example of doing this.